acceptance/phase_test.go

```diff
@@ -415,3 +415,64 @@ func assertImageOSAndArchAndCreatedAt(t *testing.T, imageName string, phaseTest
 	h.AssertEq(t, inspect.Architecture, phaseTest.targetDaemon.arch)
 	h.AssertEq(t, inspect.Created, expectedCreatedAt.Format(time.RFC3339))
 }
+
+func assertRunMetadata(t *testing.T, path string) *platform.RunMetadata { //nolint
+	contents, err := os.ReadFile(path)
+	h.AssertNil(t, err)
+	h.AssertEq(t, len(contents) > 0, true)
+
+	var runMD platform.RunMetadata
+	_, err = toml.Decode(string(contents), &runMD)
+	h.AssertNil(t, err)
+
+	return &runMD
+}
+
+func updateTOMLFixturesWithTestRegistry(t *testing.T, phaseTest *PhaseTest) { //nolint
```

Comment:
The setup for acceptance is getting a little unwieldy. We might try just mounting in the `/layers` directory instead of building it into the test container (which would actually be more realistic). But that is a bigger overhaul than I'd like to do now

Reply:
do you want to capture that as a separate #goodfirstissue? if it's just a tests refactor.  one of us might get to it or someone looking for an opportunity to contribute might.